app/services/activity/import.rb

```diff
@@ -287,6 +287,10 @@ def create
           return
         end
 
+        if @activity.third_party_project? && row["Original commitment figure"].blank?
+          @errors[:commitment] = ["", I18n.t("importer.errors.activity.commitment_required")]
+        end
```

Comment:
Again I wonder if this is a converter thing, although the previous lines are doing a similar thing

Reply:
We don't have access to `@activity` in the converter, which is why we do this outside.